letsdrill.rb

```diff
@@ -1,15 +1,29 @@
 def get_letter_grade(integer)
  
   #Put your code here!
-
+  if (integer >= 90)
+    "A"
+  elsif (integer >= 80)
+    "B"
+  elsif (integer >= 70)
+    "C"
+  elsif (integer >= 60)
+    "D"
+  else
+    "F"
+  end
```

Comment:
I like ranges here too, but this might be slightly more readable since you don't have to worry about `...` vs `..`